athena/llm_core/llm_core/loaders/model_loaders/ollama_loader.py

```
@@ -0,0 +1,62 @@
+from langchain_community.chat_models import ChatOllama
+from athena.logger import logger
+
+OLLAMA_PREFIX = "ollama_"
+
+OLLAMA_BASE_URL: str = os.getenv("OLLAMA_ENDPOINT", "http://localhost:11434")
+GPU_USER = os.getenv("GPU_USER")
+GPU_PASSWORD = os.getenv("GPU_PASSWORD")
+
+_auth = (GPU_USER, GPU_PASSWORD) if GPU_USER and GPU_PASSWORD else None
+_headers = (
+    {"Authorization": requests.auth._basic_auth_str(GPU_USER, GPU_PASSWORD)}  # type: ignore
```

Pylint (3.3.7)

[warning] Access to a protected member _basic_auth_str of a client class

(W0212)